main.go

```diff
@@ -5,8 +5,19 @@ import (
 )
 
 func main() {
-	// this is no purpose comment and should trigger a review comment
-	// fmt.Prinln("hello world")
+	// this comment doesnt add any value and should trigger a review comment instead..
```

Comment:
The comment on line 8 does not add any value and should be removed. It's better to keep the code clean and free of unnecessary comments.